examples/client_example/client_example.py

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env python
+"""Python Client Example."""
+
+# Copyright 2012 - 2017, New York University and the TUF contributors
+# SPDX-License-Identifier: MIT OR Apache-2.0
+
+import argparse
+import logging
+import os
+import shutil
+import sys
+from pathlib import Path
+
+from tuf.exceptions import RepositoryError
+from tuf.ngclient import Updater
+
+# define directory constants
+HOME_DIR = Path.home()  # user home dir
+DOWNLOAD_DIR = "./downloads"  # download dir
+METADATA_DIR = f"{HOME_DIR}/.local/share/tuf_metadata_example"  # metadata dir
+CLIENT_EXAMPLE_DIR = os.path.dirname(os.path.abspath(__file__))  # example dir
+
+
+def init():
+    """
+    The function that initializes the TUF client infrastructure.
+
+    It creates the metadata directory and downloads the ``root.json``.
+    """
```

Comment:
please review the printing in this function: I think it prints too much but more importantly the prints are incorrect (it always claims to have done a thing even if it did not do the thing at all). 

the docstring is incorrect: root.json is not downloaded, just copied from source dir (which is the correct choice in my opinion: we are giving a good example of shipping the root.json with the app)